RealmSwift/Sync.swift

```diff
@@ -389,6 +452,34 @@ public extension User {
         return ObjectiveCSupport.convert(object: config)
     }
 
+    /**
+     Create a sync configuration instance.
+
+     - parameter partitionValue: The `BSON` value the Realm is partitioned on.
+     - parameter clientResetMode: Determines file recovery behavior during a client reset.
+     - parameter notifyBeforeClientReset: A callback which notifies prior to a client reset occurring. See: `notifyBeforeClientReset`.
+     - parameter notifyAfterClientReset: A callback which notifies after a client reset has occurred. See: `notifyAfterClientReset`.
+
+
+     Additional settings can be optionally specified. Descriptions of these
+     settings follow.
+
+     `enableSSLValidation` is true by default. It can be disabled for debugging
```

Comment:
does `enableSSLValidation` exist anymore?

Reply:
I don't think it does? It was previously on the sync manager, then got moved to syncConfiguration. But I no longer see it all. The docstring must have been left over?